react/src/components/atoms/media/SiteLogo/index.js

```diff
@@ -17,13 +19,24 @@ SiteLogo.propTypes = {
   url: PropTypes.shape({
     /** The URL for the site root */
     domain: PropTypes.string
-  })
+  }),
+  image: PropTypes.shape(PropTypes.Image),
+  siteName: PropTypes.string,
+  title: PropTypes.string
 };
 
 SiteLogo.defaultProps = {
   url: {
     domain: '/'
-  }
+  },
+  image: {
+    src: logo,
+    alt: 'Massachusetts state seal',
+    width: 45,
+    height: 45
```

Comment:
@clairesunstudio the image w/h props are affecting the img tag attributes, but due to the `.ma__site-logo a img` properties, they do not have a visual affect; is that ok for now? (marking approved, otherwise).

Reply:
thanks for noting that, will follow up on it. I'm trying get a quick on in without touching the patternlab side